Big2/src/screens/HomeScreen.js

```diff
@@ -47,7 +47,9 @@ const HomeScreen = ({ navigation }) => {
       />
       <Button 
         title="SETTINGS"
-        onPress={() => navigation.navigate('Settings')}
+        onPress={() => Alert.alert("Settings coming soon")}
+        width='45%'
+        margin={10}
```

Comment:
Lol, why? Why not just navigate to `Settings`?

Reply:
Everything looks good though

Reply:
Nice work, fellas!